velox/exec/ExchangeSource.h

```diff
@@ -94,7 +95,7 @@ class ExchangeSource : public std::enable_shared_from_this<ExchangeSource> {
   // Returns runtime statistics. ExchangeSource is expected to report
   // background CPU time by including a runtime metric named
   // ExchangeClient::kBackgroundCpuTimeMs.
-  virtual folly::F14FastMap<std::string, int64_t> stats() const = 0;
+  virtual folly::F14FastMap<std::string, RuntimeCounter> stats() const = 0;
```

Comment:
This change may break Prestissimo. If so, we'll need to come up with a backwards compatible way to introduce this change.

See #5953 for an example.